spec/services/profiles/edit_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ::Profiles::EditService do
+  subject(:service) do
+    described_class.call(profile, updated_params)
+    profile.reload
+  end
+
+  let(:profile)         { create(:profile) }
+  let(:updated_surname) { 'Updated surname' }
+  let(:updated_phone)   { '0987654321' }
+
+  let(:updated_params) do
+    {
+      surname: updated_surname,
+      phone: updated_phone
+    }
+  end
+  let(:invalid_params) do
+    {
+      surname: 'a'
+    }
+  end
+
+  it 'updates params' do
+    expect { service }.to change(profile, :surname)
```

Comment:
it is enough to check if only surname or if only phone is change (it should changes both or doesn't change anything )